cargo-dist/templates/ci/github/partials/publish_homebrew.yml.j2

```
@@ -36,6 +36,9 @@
             name=$(echo "$filename" | sed "s/\.rb$//")
             version=$(echo "$release" | jq .app_version --raw-output)
 
+            brew update
+            brew style --fix "Formula/${filename}"
```

It looks like `brew style` exits non-zero if it fails to fix every possible issue with a formula; we should guard this with `|| true` to make sure that doesn't fail the build.

```suggestion
            brew style --fix "Formula/${filename}" || true
```